plugins/omar-oms-plugin/grails-app/controllers/omar/oms/ImageSpaceController.groovy

```diff
@@ -102,10 +106,10 @@ class ImageSpaceController implements AsyncController
     }
     catch ( e )
     {
-      e.printStackTrace()
+        logger.trace("There was an error in ImageSpaceController line 109", e)
```

Comment:
The logger will print the class and line number. Is there a more interesting message here?